firebase-database/src/commonTest/kotlin/dev.gitlive.firebase.database/database.kt

```diff
@@ -65,6 +65,42 @@ class FirebaseDatabaseTest {
         assertEquals(3, firebaseDatabaseChildCount)
     }
 
+    @Test
```

Comment:
_I'd appreciate some help on running JS tests for `firebase-database` module locally. there is a warning on `jsLegacyTest` run on current `master`_
```
ReferenceError: $module$firebase_compat_app is not defined
    at ..../firebase-database/src/jsMain/kotlin/dev/gitlive/firebase/database/database.kt:18:1
    at Object.<anonymous> (..../build/js/packages/firebase-kotlin-sdk-firebase-database-js-legacy-test/kotlin/firebase-kotlin-sdk-firebase-database-js-legacy-test.js:1093:2)
```
_and it seems like the tests aren't executed at all - no reports, no DB updates_